iscc_web/api/media.py

```diff
@@ -45,71 +38,48 @@ async def upload_file(self, request: Request):
         except AttributeError:
             content_type = ""
 
-        # Create ID
-        media_id = ic.Flake().string.lower()
-        file_meta = UploadMeta(
-            file_name=file_name,
-            content_type=content_type,
-            client_ip=request.client_ip,
-        )
+        # Create package directory
+        media_id, package_dir = await self.create_package()
 
-        # Save Uploaded data
-        await self.write_meta(media_id, file_meta)
-        async with aiofiles.open(self.file_path(media_id), "ab") as outf:
-            async for chunk in request.stream():
-                await outf.write(chunk)
+        # Store file metadata
+        ip = request.client_ip
```

Comment:
I want to note that an IP address is considered as personal data by GDPR, so we will most likely need to add some kind of consent to or before the upload of files if it stores the IP address.